### DISCUSSION TITLE
XBRZ 2X filter

### DISCUSSION THREAD

**Discussion by HJMBgit84** [rafaelvcaetano/melonDS-android#539] [Ideas]
Hi, 

Thanks and congratulations for the great update 1.6.0 as you are saying to request freely new filters can you please Add XBRZ 2X filter for video? :)

Also Upscaling quality if possible. Thanks for the great work Rafael 💯

**Comment by ghost** [comment]
Need OPENGL renderer first before that!

**Reply by AllwinAristo** [reply]
> Need OPENGL renderer first before that!

How to enable opengl render

**Reply by rafaelvcaetano** [reply]
Unfortunately, the OpenGL renderer is not yet implemented.

**Reply by Unknowncat1991** [reply]
> Unfortunately, the OpenGL renderer is not yet implemented.

Since now OpenGL is implented could possibly make a custom shader supports now?
I want to test all shaders from this repo https://github.com/jdgleaver/drastic_ds_shaders?tab=readme-ov-file
